packages/appsync-modelgen-plugin/src/preset.ts

```diff
@@ -217,6 +217,37 @@ const generateDartPreset = (
   return config;
 };
 
+const generateManyToManyModelStubs = (options: Types.PresetFnArgs<AppSyncModelCodeGenPresetConfig>) : TypeDefinitionNode[] => {
+  let models = new Array<TypeDefinitionNode>();
+  let manyToManySet = new Set<string>();
+  options.schema.definitions.forEach(def => {
+    if (def.kind === 'ObjectTypeDefinition') {
+      def?.fields?.forEach(field => {
+        field?.directives?.forEach(dir => {
+          if (dir?.name?.value === 'manyToMany') {
+            //manyToManySet.add(dir?.arguments?.relationName);
+            dir?.arguments?.forEach(arg => {
+              if(arg.name.value === 'relationName' && arg.value.kind === 'StringValue') {
+                manyToManySet.add(arg?.value?.value);
```

Comment:
Same here. `arg.value` is used without a null check in the previous statement. So if it is not null in the conditional statement, it can't be null in this statement.

Reply:
fixed